loadconda

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# POSIX non-interactive shells do not guarantee any file to be loaded so there
+# is no profile or rc file that we can reliably put this line into
+
+try_source() {
+    [ ! -e "$1" ] || . "$1"
+}
+
+if ! conda activate 2>/dev/null
+then
+    try_source /opt/conda/etc/profile.d/conda.sh
+    try_source "${HOME}/anaconda2/etc/profile.d/conda.sh"
+    try_source "${HOME}/miniconda/etc/profile.d/conda.sh"
+    if which conda >/dev/null 2>/dev/null
+    then
+        conda_dir=$(dirname $(dirname $(which conda)))
+        try_source "$conda_dir/etc/profile.d/conda.sh"
+    fi
+fi
```

Comment:
Exit code?

Reply:
The last command executed is the exit code of the script, so I modified `try_source` to return true if the file exists (`.` either returns successfully or exits the script immediately, so its return code is irrelevant). I chained the `try_source`s together with `||` so that one of them must succeed for `loadconda` to succeed.